models/ac_generator.cpp

```diff
@@ -202,7 +202,7 @@ nest::ac_generator::pre_run_hook()
 void
 nest::ac_generator::update( Time const& origin, const long from, const long to )
 {
-  assert( to >= 0 && ( delay ) from < kernel().connection_manager.get_min_delay() );
+  assert( to >= 0 and ( delay ) from < kernel().connection_manager.get_min_delay() );
```

Comment:
Do plan to replace C-style casts like the one here (`( delay ) from`) to C++ `static_cast`s? If not, can you do this here?

Reply:
Can create a PR where I replace those. I would rather do it once for all than just for some "random" occurrences.

Reply:
See #2506